cli/root.go

```diff
@@ -824,11 +829,15 @@ func (r *RootCmd) checkVersions(i *clibase.Invocation, client *codersdk.Client)
 	if isConnectionError(err) {
 		return nil
 	}
-
 	if err != nil {
 		return xerrors.Errorf("build info: %w", err)
 	}
 
+	dconfig, err := client.UnprivilegedDeploymentConfig(ctx)
```

Comment:
IMO you should toss the upgrade message in `buildinfo`. It's somewhat jank, but I think less so than making a new request and endpoint.